src/components/ViewingsList/utils.ts

```diff
@@ -56,6 +60,9 @@ export const i18n = {
   viewingDatesLabel: '_viewingDatesLabel_',
   viewingHours1Label: '_viewingHours1Label_',
   viewingHours2Label: '_viewingHours2Label_',
+  emailLabel: '_emailLabel_',
```

Comment:
For these i18n constants what is the purpose of the leading and trailing underscores?

Reply:
To differentiate between the normal text and a "translated" text.